README.md

```diff
@@ -3,8 +3,43 @@
 If you are the kind of person who likes thinking _long-term_ and _high-impact_
 this repo is for you!
 
-**D**o **W**hat **Y**ou **L**ove is _not_ a 3-month stepping-stone project ...
+**D**o **W**hat **Y**ou **L**ove is _not_ a stepping-stone project, we're in it for the long-haul. And after a year and a half, we're making sure that the direction we're heading in is clear to everyone who wants to know.
```

Comment:
semantic line feeds. http://rhodesmill.org/brandon/2012/one-sentence-per-line
also "year and a half"? when did you write this? we've both being dwyl-ing for a _lot_ longer than 18 months. please remove. ✂️